### ISSUE TITLE
deciding ground truth

### ISSUE THREAD

**Issue by fedhere** [fedhere/DtUhackOutliers#4]
We need to make a decision about our ground truth: we have the Tabby star, which is unusual and we would like all of our algorithm to recognize that. I think we should decide a threshold beyond which the algorithm "fails" cause it ranked the Tabby star not "unusual" enough. I think it should be minimum rank. please iterate on this. Any other objects that are acknowledged in the literature to be weird?

**Comment by juramaga** [comment]
Hi, 

I shared via email a plot showing our weirdness histogram, from URF, and where Tabby's star falls there. I attach the plot below. In numbers, there are only 4.7% of the objects (when the full Q16 of Kepler data is considered). I suggest we use the 5% mark as the threshold.

I do not know about other objects in particular, but cataclysmic events seem to be weird enough. Daniel and Lucianne identify KIC 7446357 as another weird object. Unfortunately, this source behaves weirdly in a different quarter than the one we are using (16).

Dennis and I can do a search of cataclysmic objects and other weird stuff we found. Does 5% sound good to people?

[Weirdness_Histogram_112,930_LCs.pdf](https://github.com/fedhere/DtUhackOutliers/files/2877406/Weirdness_Histogram_112.930_LCs.pdf)

**Comment by juramaga** [comment]
Some other weirdos we find in the large Kepler Dataset:

* Dwarf novas: kplr002436450, kplr007431243, kplr007446357, kplr007659570, kplr007742289, kplr008415928, kplr009778689. These are all in the top 2% of weirdness.
* Cataclysmic variables: kplr003952037, kplr006615102, kplr006691294, kplr007524178, kplr007680833, kplr008625249, kplr009071514, kplr009202990, kplr012062071. Also in the top 2%
* Low mass X-ray binary candidate [GPG2012] X1: kplr005112508. Top 1%.
* Blue straggler kplr005024084. Top 2%. Other blue stragglers do not show the same crazy LC.
* Variables of BY Dra type: kplr002437317, kplr004935181, kplr004935249, kplr004936533, kplr004937206, kplr005110407. All but one in the top 2%
*BL Lacertae objects (AGNs):  kplr007175757, kplr006690887. Top 2%
* Contact binary of AM Her type kplr004547333. Top 0.2%
* Low surface brightness galaxy MCG+07-39-011: kplr006350306. Top 0.1%
* High proper motion stars: kplr008454245, kplr006062298, kplr008381329, kplr009896438, kplr008381324, kplr009532632. Top 0.1%
* Brown dwarf with flares: kplr004996074. Top 0.05%. This one is interesting.
* White dwarves kplr008626021, kplr004357037, kplr009535405, kplr011911480
* RR lyrae stars are all in the top 2%
* QSOs kplr004843809, kplr007102641, kplr008482611, kplr010663134, kplr011606854, kplr012208602, kplr012984288. All but one in top 0.5%
* Planet candidates have all kinds of weirdnesses, but those with clear deeps in unfolded light curves have high weirdness.
* Rotationally variable kplr001576144 (Planet?)

**Comment by AshishMahabal** [comment]
Indeed, if we do not have a ground truth set in 2500, we can not "proof" our methods on that set. I thought the idea was to compare our methods (and the outliers they find). But yes, if we can use a larger dataset with known outliers, things become simpler.
(Sorry, I had not looked at the GitHb site for a long time. Kushal has newer results on the 2500 which we will be including - I will look through other "issues" to see what was decided).

**Comment by juramaga** [comment]
Thank you Ashish!

Have you been able to put the latest results in the draft? It would be great to have something in place by the end of next week, as I am traveling to Berkeley to show some of the stuff we have been doing. Cheers.
